examples/lifelong_learning/atcii/README.md

```diff
@@ -1,8 +1,8 @@
 # Using Lifelong Learning Job in Thermal Comfort Prediction Scenario
 
 This document introduces how to use lifelong learning job in thermal comfort prediction scenario. 
-Using the lifelong learning job, our application can automatically retrains, evaluates, 
-and updates models based on the data generated at the edge.
+Using the lifelong learning job, our application can automatically retrain, evaluate, 
+and update models based on the data generated at the edge.
 
 ##  Thermal Comfort Prediction Experiment
```

Comment:
please add more words to explain `Thermal Comfort Prediction` what it is, what is problem and requirements in this scenario? How to resolve it by lifelong learning?

Reply:
This is a very helpful piece of advice. Which do you think is better, quoting a detailed review or explaining it in a readme that app developers are interested in?

Reply:
write here is ok, 
BTW, the pr name is wrong, please check.

Reply:
this pr will be closed because this PR has caused confusion, stay up-to-date by following #143 and #142